libs/compute/include/hpx/compute/host/target.hpp

```diff
@@ -25,7 +25,7 @@
 
 namespace hpx { namespace compute { namespace host
 {
-    struct target
+    struct HPX_EXPORT target
```

Comment:
If we remove the `HPX_EXPORT` here we will have to add it at least for the serialization functions below.

Reply:
Note: I've *added* the `HPX_EXPORT`. I could remove it from `native_handle_type` below though...

Reply:
Uhh, sorry - I'm still sleeping, apparently… And yes, we don't need it for the `native_handle_type`